.travis.yml

```diff
@@ -1,15 +1,19 @@
 language: ruby
+cache:
+  directories:
+    - $HOME/.gem/ruby
+    - Library/Homebrew/vendor/bundle
 
 matrix:
   include:
     - os: osx
-      osx_image: xcode8.1
+      osx_image: xcode8.3
       rvm: system
     - os: linux
       rvm: 2.0.0
 
 before_install:
-  - export HOMEBREW_DEVELOPER="1"
+  - export HOMEBREW_DEVELOPER=1
```

Comment:
> Any other thoughts you have on improving our caching are super welcome, by the way.

Nothing I can think of.

> Then Travis shows it in the UI and I get upset because I'm weird 😆.

Ah, right, I know exactly what you mean! 😂